pkg/apis/metal3/v1alpha1/baremetalhost_types.go

```diff
@@ -451,6 +454,7 @@ type ProvisionStatus struct {
 // +kubebuilder:printcolumn:name="Provisioning Status",type="string",JSONPath=".status.provisioning.state",description="Provisioning status"
 // +kubebuilder:printcolumn:name="Consumer",type="string",JSONPath=".spec.consumerRef.name",description="Consumer using this host"
 // +kubebuilder:printcolumn:name="BMC",type="string",JSONPath=".spec.bmc.address",description="Address of management controller"
+// +kubebuilder:printcolumn:name="DisableCertificateVerification",type="bool",JSONPath=".spec.bmc.disableCertificateVerification",description="Skip certificate validation when true"
```

Comment:
I'm not wholly convinced this is one of the top 8 most important columns we need to see in `kubectl get bmh`.

Reply:
Fixed